package.json

```diff
@@ -20,7 +20,7 @@
     "blc:suspensive.org": "pnpm blc https://suspensive.org --ro --exclude='https://x.com/*' --exclude='https://codesandbox.io/api/v1/sandboxes/define?undefined&environment=create-react-app'",
     "build": "turbo run build",
     "changeset": "changeset",
-    "changeset:publish": "pnpm run prepack && changeset publish && pnpm run jsr:dry-run && pnpm run jsr:publish",
```

Comment:
<img width="903" alt="image" src="https://github.com/user-attachments/assets/d54c2fc6-6aba-4632-bd5a-b611915bb8d3" />

At the time of running dry-run, jsr will not have deployed @suspensive/react-query's dependencies, @suspensive/react-query-*, so remove dry-run from changeset:publish.